### PR TITLE
Added types for wink-jaro-distance

### PR DESCRIPTION
**Pull request by Wazbat**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/wink-jaro-distance/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Type definitions for wink-jaro-distance 2.0
+// Project: https://github.com/winkjs/wink-jaro-distance (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```
Comment (MysteryBlokHed) [inline types/wink-jaro-distance/index.d.ts:2]:
```suggestion
// Project: https://github.com/winkjs/wink-jaro-distance
```

File: types/wink-jaro-distance/wink-jaro-distance-tests.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import jaro = require('wink-jaro-distance');
+
+const result = jaro('abcdef', 'fedcba');
+const result2 = jaro('abcdef', 'fedcba');
+
+// Just need to make sure the types are a number I suppose?
```
Comment (MysteryBlokHed) [inline types/wink-jaro-distance/wink-jaro-distance-tests.ts:6]: You can assert types with the `$ExpectType` dtslint feature instead:
```typescript
result.distance; // $ExpectType number
result.similarity; // $ExpectType number
```

**Review by MysteryBlokHed — CHANGES_REQUESTED** [review]

File: types/wink-jaro-distance/index.d.ts (hunk 1 above)
Comment (Wazbat) [reply types/wink-jaro-distance/index.d.ts]: My bad! I'll go ahead and remove that

File: types/wink-jaro-distance/wink-jaro-distance-tests.ts (hunk 2 above)
Comment (Wazbat) [reply types/wink-jaro-distance/wink-jaro-distance-tests.ts]: Oh perfect! I wasn't ware of those features. I'm honestly really really impressed with dtslint. Especially with how it checks the original source code and things

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Review by gabritto — APPROVED** [review]

**Comment by Wazbat** [comment]
Ready to merge

**Pull request merged**